cclib/parser/nwchemparser.py

```diff
@@ -1261,6 +1295,22 @@ def extract(self, inputfile, line):
                 self.append_attribute("vibirs", float(line.split()[5]))
                 line = next(inputfile)  # next line
             self.set_attribute("vibfreqs", vibfreqs)
+
+        # Grab rotational constants (convert cm-1 to Hz)
+        if line.strip().startswith('A='):
+            roconst, rotemp = [], []
+            split_line = line.split()
+            roconst.append(utils.convertor(float(split_line[1]),'wavenumber','Hz'))
+            rotemp.append(float(split_line)[4])
+            line=next(inputfile)
+            while line.strip().startswith('B=') or line.strip().startswith('C='):
+                split_line = line.split()
+                roconst.append(utils.convertor(float(split_line[1]),'wavenumber','Hz'))
+                rotemp.append(float(split_line)[4])
+                line=next(inputfile)
+            self.set_attribute('rotconsts', roconst)
+            self.set_attribute('rottemp', rotemp)

```

Comment:
What do we think about rotational temperature? https://github.com/cclib/cclib/issues/1093#issuecomment-1018112286 definitely a method in the future, but should we parse it too?

Reply:
Hm, if they are available, having them immediately available and not have to do extra work of a recalculation through a method seems like a benefit to me, but maybe I am not seeing a reason that this could be problematic?